src/pages/Shop.js

```diff
@@ -1,21 +1,84 @@
-import React from 'react';
-import '../style/main.scss';
-import { createGlobalStyle } from 'styled-components'
+import React from 'react'
+import '../style/main.scss'
+import styled, { createGlobalStyle } from 'styled-components'
+import { Composition } from 'atomic-layout'
+import { Pictures } from '../components/Pictures'
 
 const GlobalStyle = createGlobalStyle`
   body {
     background-color: ${props => props.bgColor};
   }
 `
 
+const areasMobile = `
+  contentLeft
+  image
+  contentRight
+`
+const areasLarge = `
+  contentLeft image contentRight
+  / 1fr 60% 1fr
+`
+
+const pictures = [
+  {
+    src:
+      'https://cdn.dribbble.com/users/59947/screenshots/6133129/shipwreck_island_dribbble.jpg'
+  },
+  {
+    src:
+      'https://cdn.dribbble.com/users/2761/screenshots/3692195/breath-of-the-wild-merge-4.png'
+  },
+  {
+    src: 'https://cdn.dribbble.com/users/2407/screenshots/2679126/greedo.png'
+  },
+  {
+    src:
+      'https://cdn.dribbble.com/users/1355613/screenshots/6441984/landscape.jpg'
+  },
+  {
+    src:
+      'https://cdn.dribbble.com/users/1355613/screenshots/6248741/mr._duck_in_the_rain.jpg'
+  },
+  {
+    src:
+      'https://cdn.dribbble.com/users/1355613/screenshots/6284276/attachments/1346008/customer_tags_2.png'
+  }
+]
+
+const index = pictures.length / 2
```

Comment:
Try to have an odd number of `pictures` (for example 7 pictures).
Then this division by 2 will not work properly.

You need to _round_ the number to the closest smallest or highest:

```js
// pictures.length is 7
const index = pictures.length / 2 // 3.5 - that's not okay, need a whole number
const index = Math.ceil(pictures.length / 2) // 4 - rouding to the highest: 3.5 -> 4
const index = Math.floor(pictures.length / 2) // 3 - rounding to the lowest: 3.5 - > 3
```